lib/publiccloud/provider.pm

```diff
@@ -379,12 +379,13 @@ sub vault_api {
     my $data   = $args{data}   // {};
     my $ua     = Mojo::UserAgent->new;
     my $url    = get_required_var('_SECRET_PUBLIC_CLOUD_REST_URL');
+    my $namespace = get_var('PUBLIC_CLOUD_VAULT_NAMESPACE', '');
```

Comment:
I thought we are hiding everything related to Vault from UI . I understand that this is not something which allow to break through but still I think we should keep same pattern

Reply:
For me it is actually quite useful to see this, at it makes transparent which credentials was used.
In deed it break the pattern, the reason is, it isn't a worker related variable but a `isos post` related one.

Reply:
We discussed this with @cfconrad and we think that this part is not very secret and in fact I think it could be good to quickly check if test is using a different namespace, as configuration can be different.

But I'm OK if most of you think that we have to keep it secret (my first variable name was `_SECRET_PUBLIC_CLOUD_NAMESPACE` :D).

Reply:
fine for me to keep it visible just wanted to mention that it breaks the pattern